test/CMakeLists.txt

```diff
@@ -15,6 +15,17 @@
 set (ranlux_24_48_test.pass_timeout_debug "900") # 15min
 set (ranlux_24_48_test.pass_timeout_release "720") # 12min
 
+include(CheckIncludeFileCXX)
+CHECK_INCLUDE_FILE_CXX("tbb/version.h" HAVE_TBB)
```

Comment:
Why `version.h` but not `tbb.h`? I.e. `oneapi/tbb.h`? Or we should consider not only oneAPI TBB distributions?
What if TBB 2020 or older is available in environment?

Reply:
This becomes relevant if we follow a common CMake approach: inclusion of  a library through `find_package`.